src/yadism/coefficient_functions/coupling_constants.py

```diff
@@ -247,10 +299,22 @@ def propagator_factor(self, mode, Q2):
             * (1.0 - self.theory_config["sin2theta_weak"])
         )
         eta_phZ /= 1 - self.obs_config["propagatorCorrection"]
+
+        # Need to specify Wilson coefficient?
+        C_4F = 1/4
+        # Modify with more precise value
+        alpha = 1 / 137
+        # Should get it from param card
+        BSM_scale = 1000
+        eta_ph4F = ((C_4F) / (4 * np.pi*alpha)) *(Q2/BSM_scale**2)
```

Comment:
I guess, all three parameters should come from the outside? Then I wonder if we should condense them all into one $P^2 = \Lambda^2 \cdot 4\pi/C_{4F}$ (the name is random) such that `eta_ph4f = Q2/P2`

Reply:
In practice there is a redundancy between `C_{4F}` and `\Lamdba` since we only consider dimension-6 operators, so only `\Lambda` would typically be defined in the runcard.
It could make sense to condense the prefactor `P = C_{4F} / (4 * \pi * \alpha)` you are right but the ratio `Q2 / \Lamdba^2` is the kind of thing that is intuitive to see I think

Reply:
I would say this depends if there is another dependency on either of these things - if there is not, I think we should collapse them to avoid an ambiguous configuration